app/controllers/application_controller.rb

```diff
@@ -71,7 +71,7 @@ def prepare_catalog
     end
 
     if USE_NEWS_FEED
-      page = Nokogiri::HTML(open("http://www.sparcrequestblog.com"))
+      page = Nokogiri::HTML(open("https://www.sparcrequestblog.com"))
```

Comment:
This change is also occurring in PR - KG - Display SRID When Managing Service Request in Catalog #341